test/readme.md

```diff
@@ -26,6 +26,18 @@ each spec, a new ssh connection will be open to the virtual server. Mainly this
 connections are wrapped in some helpers, and a Node attribute is exported in
 case that we need to run a specific command on a node.
 
+
+## Requirements
+
+Before run any test, you should have the following tools installed.
+
+- Virtualbox
```

Comment:
Specify which versions of these dependencies should be installed. 
On the Cilium build slaves, we have:
* VirtualBox 5.1
* Vagrant 2.0.0

Version of Docker, Docker Compose, Gingko, and Gomega and golang are not currently specified for the build slaves. Is go 1.9 alright? I can take care of this as part of https://github.com/cilium/cilium/issues/1768 .